C-Plus-Plus/other/Longest_subarray_with_equal_0_1.cpp

```diff
@@ -0,0 +1,102 @@
+/**
+*Author-:Palak
```

Comment:
The Git Commit would be forever credited to you. It would be good to remove the author name